### PR TITLE
Split Free storage payment between read and write pool.

### PR DESCRIPTION
**Pull request by Sriep**
Creates a new sc.yaml entry `read_pool_fraction`
On free storage allocation split the free funds between an allocation write pool and read pool in the ratio of `read_pool_fraction: 1-read_pool_fraction`.
Update unit test.

### CODE REVIEW

File: docker.local/config/0chain.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 1.0
 
 logging:
   level: "debug"
-  console: false # printing log to console is only supported in development mode
+  console: true # printing log to console is only supported in development mode
```
Comment (moldis) [inline docker.local/config/0chain.yaml:5]: I guess it's accidentally here

File: docker.local/config/0chain.yaml [hunk 2]
```diff
@@ -117,10 +117,10 @@ server_chain:
     max_simultaneous_from_sharders: 30
 
 network:
-  magic_block_file: config/b0magicBlock_4_miners_2_sharders.json
+  magic_block_file: config/b0magicBlock_4_miners_1_sharder_piers.json
```
Comment (moldis) [inline docker.local/config/0chain.yaml:120]: +1 accidentally added here

File: docker.local/config/0chain.yaml [hunk 3]
```diff
@@ -117,10 +117,10 @@ server_chain:
...
   initial_states: config/initial_state.yaml
   genesis_dkg: 0
-  dns_url: "" # http://198.18.0.98:9091
+  dns_url: "" # http://198.18.0.98:9091 http://192.168.1.100:9091
```
Comment (moldis) [inline docker.local/config/0chain.yaml:123]: +1 accidentally added here

File: docker.local/config/sc.yaml [hunk 4]
```diff
@@ -151,10 +154,10 @@ smart_contracts:
     block_reward:
       block_reward: 1
       qualifying_stake: 1
-      sharder_ratio: 80
-      miner_ratio: 20
-      blobber_capacity_ratio: 20
-      blobber_usage_ratio: 80
+      sharder_ratio: 10
```
Comment (moldis) [inline docker.local/config/sc.yaml:157]: +1 accidentally added here

**Review by moldis — CHANGES_REQUESTED** [review]
Minor cleanup

File: docker.local/config/sc.yaml (hunk 4 above)
Comment (Sriep) [reply docker.local/config/sc.yaml:157]: No, actually that one is actually part of the PR. Change the block reward rations.

File: docker.local/config/0chain.yaml [hunk 5]
```diff
@@ -117,7 +117,7 @@ server_chain:
     max_simultaneous_from_sharders: 30
 
 network:
-  magic_block_file: config/b0magicBlock_4_miners_2_sharders.json
+  magic_block_file: config/b0magicBlock_4_miners_2_sharder_piers.json
```
Comment (moldis) [inline docker.local/config/0chain.yaml:120]: I think this still wrong as pointing to ...piers.json

**Pull request merged**